src/ansys/dpf/core/fields_container.py

```diff
@@ -590,6 +585,118 @@ def animate(self, save_as=None, deform_by=None, scale_factor=1.0, **kwargs):
             **kwargs,
         )
 
+    def animate_modal(
```

Comment:
@clatapie We try to not add too many new APIs in the fieldscontainer. Could you instead create a new field: animator.py and create in it a function def animate_mode(fields_container, mode_number, save_as=None, deform_by=None, frame_number=21, scale_factor=1.0, **kwargs) please?
For the implementation, I'll let @PProfizi review